tests/service_urls/test_settings.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import LazySettings, Settings
+from django.test import TestCase
+
+settings = LazySettings()
+settings.configure(Settings('tests.service_urls.settings'))
+
+
+class SettingsTest(TestCase):
+    def test_databases(self):
+        result = settings.DATABASES
+        self.assertEqual(result['default']['ENGINE'], 'django.db.backends.sqlite3')
+        self.assertEqual(result['default']['NAME'], ':memory:')
+        self.assertEqual(result['postgresql']['ENGINE'], 'django.db.backends.postgresql')
+        self.assertEqual(result['postgresql']['NAME'], 'd8r82722r2kuvn')
+        self.assertEqual(result['postgresql']['HOST'], '')
+        self.assertEqual(result['postgresql']['USER'], 'uf07k1i6d8ia0v')
+        self.assertEqual(result['postgresql']['PASSWORD'], '')
+        self.assertEqual(result['postgresql']['PORT'], 5435)
+        self.assertEqual(result['mysql']['ENGINE'], 'django.db.backends.mysql')
+        self.assertEqual(result['mysql']['NAME'], 'd8r82722r2kuvn')
+        self.assertEqual(result['mysql']['HOST'], 'ec2-107-21-253-135.compute-1.amazonaws.com')
+        self.assertEqual(result['mysql']['USER'], 'uf07k1i6d8ia0v')
+        self.assertEqual(result['mysql']['PASSWORD'], 'wegauwhgeuioweg')
+        self.assertEqual(result['mysql']['PORT'], 3306)
```

Comment:
Not sure but they might require tests for all backends that this patch supports, and then refactor the said tests.